### ISSUE TITLE
Multiple selection in FileSystem is canceled when saving the scene

### ISSUE THREAD

**Issue by me2beats** [godotengine/godot#46745]
**Godot version:**

tested on 3.2.3, 3.2.4rc3

**OS/device including version:**

Linux

**Issue description:**

Selection of multiple files is canceled when saving the scene (using `Ctrl+S` for example).

**Steps to reproduce:**
1. Open/create a project, open/create a scene.
2. Select several files in Godot editor FileSystem dock
3. Press `Ctrl+S` or `Scene`-`Save Scene`
As you can see, only one file is selected now.

**Minimal reproduction project:**

**Comment by snougo** [comment]
yes, only one is selected, but I'm not sure this is a issue or not

**Comment by me2beats** [comment]
If cancelling the selection is a feature, when might it be useful?
Imo his is unexpected behavior anyway.
If you select many files, to do something with them but then decide to save the scene, you need to select them again, this is annoying.
